Tests/UnitsTests/UnitTests.swift

```diff
@@ -7,6 +7,8 @@ final class UnitTests: XCTestCase {
         XCTAssertNotEqual(Unit.meter, Unit.foot)
         XCTAssertEqual(Unit.meter * Unit.second / Unit.second, Unit.meter)
         XCTAssertNotEqual(Unit.newton, Unit.kilogram * Unit.meter / Unit.second.pow(2))
+        XCTAssertNotEqual(Unit.meter.pow(1.over(2)), Unit.kilogram * Unit.meter / Unit.second.pow(2))
+        XCTAssertEqual(Unit.meter.pow(1.over(6)), Unit.meter.pow(1.over(2)).pow(1.over(3)) )
```

Comment:
These tests are great, but could we also add specific tests for the Fraction functions?

Reply:
Yeah. Just haven't yet cuz I didn't want to spend more time on it if it wasn't a productive direction to go. I'll make a more robust test suite for the Fraction.

Reply:
I've added tests.